src/app/static/src/app/components/plots/bar/utils.ts

```diff
@@ -177,29 +180,30 @@ export const getErrorLineAnnotations = function(chartData: BarChartData,
             */
             const errorBarWidth = (numOfBars / (numOfBars + 10)) * halfBarWidth * 0.3;
 
-            if (errorBarData[xLabel].minus && errorBarData[xLabel].plus) {
+            const { minus, plus } = errorBarData[xLabel];
+            if (minus && plus) {
```

Comment:
for some reason typescript could not figure out that we were checking for minus and plus properties in this if statement already, had to define them as variables and then it worked? this one is a mystery to me